gcore/rawdataset.h

```diff
@@ -61,9 +61,11 @@ class CPL_DLL RawDataset : public GDALPamDataset
          virtual ~RawDataset() = 0;
 
     bool GetRawBinaryLayout(GDALDataset::RawBinaryLayout&) override;
-
+    void resetCache(void) {this->cachedCPLOneBigReadOption = std::make_pair(false, 0);}
```

Comment:
I feel a better name than resetCache() should be found, as there is a risk of confusion with the block cache.
ClearCachedConfigOption() maybe ?